style.css

```diff
@@ -0,0 +1,478 @@
+body {
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden;
+}
+
+main {
+  width: 95vw;
+  margin: 0 auto;
+  margin-top: 30px;
+}
+
+.mobile-log {
+  display: none;
+}
+
+.logo-nav {
+  background-color: #f72210;
+  padding: 0 20px;
+  width: 95vw;
+  margin: 0 auto;
+  top: 0;
+  z-index: 10;
+}
+
+.logo-nav a {
+  text-decoration: none;
+  color: white;
+}
+
+.logo-nav .brand-name {
+  font-size: 4em;
+  font-family: 'Merriweather', serif;
+  font-weight: 900;
+}
+
+.logo-nav time {
+  color: white;
+  margin-top: 50px;
+}
+
+.logo-nav .login {
+  font-weight: 600;
+  font-size: 1.1em;
+}
+
+.logo-nav .subscribe {
+  border-radius: 100px;
+  font-weight: 700;
+  padding: 3px 20px;
+  color: black;
+}
+
+.user {
+  display: none;
+}
+
+.hamburger {
+  display: none;
+  color: #e9e9e9;
+  opacity: 0.6;
+  font-size: 1.1em;
+  padding-left: 10px;
+}
+
+.right-side-bar-text-card a {
+  text-decoration: none;
+  color: black;
+}
+
+.middle-nav {
+  margin: 0 35px;
+  border-bottom: 1px solid lightgray;
+}
+
+.middle-nav ul li {
+  border-right: 1px solid lightgray;
+  padding: 0 10px;
+  height: 30px;
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.sponsore-card a {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+}
+
+.middle-nav ul li a {
+  color: black;
+  font-weight: 700;
+  font-size: 1em;
+}
+
+.middle-nav ul .search {
+  border: none;
+}
+
+.last-nav {
+  padding: 0 0 0 60px;
+}
+
+.last-nav ul li {
+  padding: 0 10px;
+  height: 30px;
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
```

Comment:
- [ ] Use bootstrap classes for these element.

I think that you get the idea that the CSS stylesheet has to ve reviews and replace all CSS rules that have a Bootstrap class... How would you know wich one? Go to the framework documentation and use the search form to look for the classes.